e2e/patches/nan+2.22.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/nan/nan_scriptorigin.h b/node_modules/nan/nan_scriptorigin.h
+index 04e55cf..9949631 100644
+--- a/node_modules/nan/nan_scriptorigin.h
++++ b/node_modules/nan/nan_scriptorigin.h
+@@ -13,7 +13,7 @@ class ScriptOrigin : public v8::ScriptOrigin {
+  public:
+ 
+ #if defined(V8_MAJOR_VERSION) && (V8_MAJOR_VERSION > 11 \
+-    && defined(V8_MINOR_VERSION) && V8_MINOR_VERSION > 7)
++    && defined(V8_MINOR_VERSION) && (V8_MAJOR_VERSION > 12 || (V8_MAJOR_VERSION == 11 && V8_MINOR_VERSION > 7)))
```

Comment:
The most recent Electron version bumped V8 to version 13, but that hasn't been updated in `nan` yet. A PR to do so is already open: https://github.com/nodejs/nan/pull/979